rplidar_composition:
  ros_parameters:
    serial_port: '/dev/serial/by-id/usb-Silicon_Labs_CP2102_USB_to_UART_Bridge_Controller_0001-if00-port0'
    serial_baudrate: '115200'
    angle_compensate: 'true'
    frame_id: 'laser_frame'